dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/MockCacheableRegistryImpl.java

```diff
@@ -17,13 +17,21 @@
 package org.apache.dubbo.registry;
 
 import org.apache.dubbo.common.URL;
+import org.apache.dubbo.common.url.component.ServiceAddressURL;
+import org.apache.dubbo.common.url.component.URLAddress;
+import org.apache.dubbo.common.url.component.URLParam;
+import org.apache.dubbo.common.utils.UrlUtils;
 import org.apache.dubbo.registry.support.CacheableFailbackRegistry;
 
+import java.util.*;
```

Comment:
please install codestyle template in your ide

you can follow this https://github.com/apache/dubbo/blob/master/CONTRIBUTING.md#code-style

import classes with wildcard is not allow in Dubbo